src/gam.py

```diff
@@ -8454,6 +8454,9 @@ def doGetNotifications():
     print u'--------------'
     print u''
 
+def orgUnitPathQuery(path):
+  return u"orgUnitPath='{0}'".format(path.replace(u"'", u"\'")) if path != u'/' else None
```

Comment:
If this is going to be it's own function let's make it more readable on multiple lines...